### PR TITLE
Added Logo1 #61

### LINKED ISSUE #61: Add logo to website
Adding finalized logo to woodog website.

### PR DESCRIPTION
**Pull request by ishitakapoor26**
Fixes #61 

## Checklist:

- [x] Have you checked there aren't other open [Pull Requests](https://github.com/Feminine-Divine/Woodog/pulls) for the same update/change?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

---

### CODE REVIEW

**Comment by ishitakapoor26** [comment]
@hariramjp777 pls review pr.

**Review by khushishikhu — APPROVED** [review]

**Comment by Aaishpra** [comment]
Can you add a screenshot for the change @ishitakapoor26 then it will be easier for us to review.

File: Woodog/templates/partials/base.html [hunk 1]
```diff
@@ -118,7 +118,7 @@
 
 <nav id="header" class="navbar navbar-expand-lg">
   <div class="container-fluid">
-    <a class="navbar-brand logo" href="#"><img src="{% static 'img/woodog-logo.svg' %}"></a>
+    <a class="navbar-brand logo" href="#"><img src="{% static 'img/woodog-logo1.png' %}"></a>
```
Comment (hariramjp777) [inline Woodog/templates/partials/base.html:121]: Hi @ishitakapoor26, Thanks for your contribution.
Can you please remove the background of the logo? It'll be good if we just have the icon and the text 'Woodog'.

**Review by hariramjp777 — CHANGES_REQUESTED** [review]

File: Woodog/templates/partials/base.html (hunk 1 above)
Comment (hariramjp777) [reply Woodog/templates/partials/base.html:121]: image

Here's the screenshot,

Comment (hariramjp777) [reply Woodog/templates/partials/base.html:121]: Also, Add `style` attribute for `` to give width.

```html
<img src="/static/img/woodog-logo1.png" style="width: 80px;">
```

File: Woodog/templates/partials/base.html [hunk 2]
```diff
@@ -118,7 +118,7 @@
 
 <nav id="header" class="navbar navbar-expand-lg">
   <div class="container-fluid">
-    <a class="navbar-brand logo" href="#"><img src="{% static 'img/woodog-logo1.png' %}"></a>
+    <a class="navbar-brand logo" href="#"><img src="{% static 'img/woodog-logo1.png' %}" style="width: 80px;"></a>
```
Comment (hariramjp777) [inline Woodog/templates/partials/base.html:121]: :+1:

**Review by hariramjp777 — APPROVED** [review]

**Pull request merged**